charts/studio/values.yaml

```
@@ -68,6 +68,11 @@ global:
     # -- Celery result URL
     resultBackend: ""
 
+  # -- Studio: Settings related to DVCX
+  dvcx: {}
+    # -- Place where DVC^X will store its temporary files
```

```suggestion
    # -- Place where DVCX will store its temporary files
```

Same as Oded's comment from before